Formula/libsequence.rb

```diff
@@ -13,10 +13,6 @@ class Libsequence < Formula
     sha256 "28860452fe218465728bc696682b9a434e9944e83431f91469afc710cb4dc089" => :x86_64_linux
   end
 
-  depends_on "boost"
-  depends_on "gsl"
-  depends_on "tbb"
-
```

Comment:
What happened to these dependencies? Are they vendored in now?

Reply:
Removed since [1.9.4](https://github.com/molpopgen/libsequence/releases/tag/1.9.4). That was the motivation of my old PR #535 .